### DISCUSSION TITLE
Add comments to the XML

### DISCUSSION THREAD

**Discussion by arthurkirkosa** [spatie/array-to-xml#223] [Q&A]
Hi,

Is there a way to add xml type comments in the generated file? 

Either when generating from array or after the fact.

Thanks

**Comment by dimitri-koenig** [comment]
Hopefully my PR #244 will be approved and then we have comments :-)

**Reply by dimitri-koenig** [reply]
Yeah, it got approved and merged. Now we have comments :-)
